src/api/app/services/gitea_status_reporter.rb

```diff
@@ -34,6 +34,9 @@ def status_options
     if @initial_report
       { context: 'OBS SCM/CI Workflow Integration started',
         target_url: Rails.application.routes.url_helpers.token_workflow_run_url(@workflow_run.token_id, @workflow_run.id, host: Configuration.obs_url) }
+    elsif @event_payload[:number]
```

Comment:
If we ever change the content of our event payloads this will break. Can we make this a bit more explicit?

Reply:
I check now the event type, this should be more reliable